cmake/FindDummy.cmake

```diff
@@ -0,0 +1,102 @@
+#####################
+## FindDummy.cmake
+## Top contributors (to current version):
+##   Gereon Kremer
+## This file is part of the CVC4 project.
+## Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+## in the top-level source directory and their institutional affiliations.
+## All rights reserved.  See the file COPYING in the top-level source
+## directory for licensing information.
+##
+# This file serves as a template for how the FindX.cmake scripts should
+# work.
+#
+# Find Dummy library
+# Exported variables:
+#   Dummy_FOUND - we have the Dummy lib, always true
+#   Dummy_FOUND_SYSTEM - we use the system version
+# Exported targets:
+#   Dummy - an imported library target
+
+# provides some utility definitions
+include(deps-helper)
+
+# initial setup
+find_package(Java REQUIRED)
+include(UseJava)
```

Comment:
I'd not include this in the helper (I fear that this will be copied all over the place even though Java will not be needed ;-)).

Reply:
Good point!